lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -103,23 +111,24 @@ internal sealed class IMRUDriver<TMapInput, TMapOutput, TResult> : IObserver<IDr
             _memoryPerMapper = memoryPerMapper;
             _memoryForUpdateTask = memoryForUpdateTask;
             _perMapperConfigs = perMapperConfigs;
-            _completedTasks = new ConcurrentBag<ICompletedTask>();
+            _totalMappers = dataSet.Count;
+            _completedTasks = new List<ICompletedTask>();
+
             _allowedFailedEvaluators = (int)(failedEvaluatorsFraction * dataSet.Count);
```

Comment:
This rounds down. So if `failedEvaluatorsFraction * dataSet.Count` is less than `1`, it means that no Evaluator is allowed to fail. Is this documented where the `failedEvaluatorsFraction` is defined?